build/release_specs/deb-rpm-rbc-spec.json

```
@@ -0,0 +1,12 @@
+{
+  "files": [
+    {
+      "pattern": "ecosys-jfrog-debs/pool/jfrog-cli-v2/(jfrog-cli-${VERSION}.*.deb)",
+      "target": "jfrog-debs/pool/jfrog-cli-v2/{1}"
```

Please also notice that this is not a Regex, but Wildcard (Use `*` instead of `.*`)
```suggestion
      "pattern": "ecosys-jfrog-debs/(pool/jfrog-cli-v2/jfrog-cli-${VERSION}*.deb)",
      "target": "jfrog-debs/{1}"
```